src/test/java/persistence/reflection/ReflectionTest.java

```diff
@@ -0,0 +1,107 @@
+package persistence.reflection;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import persistence.study.Car;
+import persistence.study.PrintView;
+
+import java.lang.reflect.*;
+import java.util.Arrays;
+import java.util.Objects;
+
+import static org.assertj.core.api.Assertions.*;
+
+
+public class ReflectionTest {
+    private static final org.slf4j.Logger log = org.slf4j.LoggerFactory.getLogger(ReflectionTest.class);
```

Comment:
```suggestion
    private static final Logger log = LoggerFactory.getLogger(ReflectionTest.class);
```
path 명시는 불필요해보여 import 해도 될 것 같아요 :)